Makefile

```diff
@@ -153,7 +153,7 @@ GATEKEEPER_MANIFEST_DIR ?= config/gatekeeper
 import-manifests: kustomize ## Import Gatekeeper manifests.
 	if [ "$${IMPORT_MANIFESTS_PATH#https://}" != "$(IMPORT_MANIFESTS_PATH)" ]; then \
 		git clone --depth=1 --branch v$(GATEKEEPER_VERSION)  $(IMPORT_MANIFESTS_PATH) $(TMP_IMPORT_MANIFESTS_PATH) ; \
-		cd $(TMP_IMPORT_MANIFESTS_PATH) && make patch-image ; \
+		cd $(TMP_IMPORT_MANIFESTS_PATH) && make patch-image && $(SED) -i 's/openpolicyagent/quay.io\/gatekeeper/g' config/manager/manager.yaml ; \
```

Comment:
Thinking more about this, this image update should be handled in `stolostron/gatekeeper` and then we just update the `IMPORT_MANIFESTS_PATH` here.

Reply:
so we need to direct IMPORT_MANIFESTS_PAT=https://github.com/open-policy-agent/gatekeeper  to  https://github.com/stolostron/gatekeeper after we change the https://github.com/stolostron/gatekeeper

Reply:
Yes, that's what I'm thinking.